maps/templates/distress/iac.dmm

```diff
@@ -0,0 +1,137 @@
+/obj/item/reagent_containers/food/snacks/skrellsnacks,
+/obj/item/reagent_containers/food/snacks/skrellsnacks,
+/obj/item/reagent_containers/food/snacks/skrellsnacks,
+/obj/item/reagent_containers/food/snacks/skrellsnacks,
+/obj/item/reagent_containers/food/snacks/nathisnack,
+/obj/item/reagent_containers/food/snacks/nathisnack,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/item/reagent_containers/food/drinks/cans/waterbottle,
+/obj/random/plushie,
+/obj/random/plushie,
+/turf/unsimulated/floor{
+	icon_state = "platebot"
+	},
+/area/centcom/distress_prep)
+"f" = (
+/obj/structure/closet/crate/medical{
+	layer = 2.99
+	},
+/obj/item/cane/crutch,
+/obj/item/cane/crutch,
+/obj/item/cane/crutch,
+/obj/item/cane/crutch,
+/obj/item/roller,
+/obj/item/roller,
+/obj/item/roller,
+/obj/item/roller,
+/obj/item/roller,
+/obj/item/roller,
+/obj/item/roller,
+/obj/item/roller,
+/obj/item/storage/box/masks,
+/obj/item/storage/box/gloves,
+/obj/item/storage/box/bodybags,
+/obj/item/storage/firstaid/adv,
+/obj/item/storage/firstaid/adv,
+/obj/item/storage/firstaid/regular,
+/obj/item/storage/firstaid/regular,
+/obj/item/storage/firstaid/o2{
+	pixel_x = -3;
+	pixel_y = -1
+	},
+/obj/item/storage/firstaid/fire{
+	pixel_x = 2;
+	pixel_y = 6
+	},
+/obj/item/storage/firstaid/toxin{
+	pixel_x = 6
+	},
+/obj/item/stack/material/steel{
+	amount = 50
+	},
+/obj/item/stack/material/plastic{
+	amount = 50
+	},
+/turf/unsimulated/floor{
+	icon_state = "platebot"
+	},
+/area/centcom/distress_prep)
+"r" = (
+/mob/living/heavy_vehicle/premade/light/iac,
+/turf/unsimulated/floor{
+	icon_state = "platebot"
+	},
+/area/centcom/distress_prep)
+"S" = (
+/obj/structure/closet/crate/freezer{
+	layer = 2.99;
+	name = "blood, surgery equipment"
+	},
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/blood/OMinus,
+/obj/item/reagent_containers/spray/cleaner,
+/obj/item/deployable_kit/surgery_table,
+/obj/item/deployable_kit/surgery_table,
+/obj/item/storage/firstaid/surgery,
+/obj/item/storage/firstaid/surgery,
+/obj/machinery/iv_drip{
```

Comment:
Is it intentional that these are different from normal iv_drips and you won't be able to push them?

Reply:
It's intentional, as the ship is small they'll be easier to load onboard. IVs are rarely pushed anyways and mostly pulled.